include/nfc/ndef/msg_parser.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef NFC_NDEF_MSG_PARSER_H_
+#define NFC_NDEF_MSG_PARSER_H_
+
+/**
+ * @file
+ * @defgroup nfc_ndef_msg_parser Parser for NDEF messages
+ * @{
+ * @brief Parser for NFC NDEF messages and records.
+ */
+
+#include <stdint.h>
+#include <zephyr/types.h>
+#include "msg_parser_local.h"
```

Comment:
Can we eliminate this include?

Local headers should not be exposed in the API (even indirectly like in Bluetooth subsys in Zephyr). You could move the necessary structures here from the local header.

Reply:
Yes, It is possible. Done